src/templates/rust-all/Makefile.tftpl

```diff
@@ -51,8 +53,10 @@ help:
 	@echo "  $(BOLD)build$(SGR0)       -- Run 'cargo build'"
 	@echo "  $(BOLD)release$(SGR0)     -- Run 'cargo build --release --target RELEASE_TARGET'"
 	@echo "                 (RELEASE_TARGET=$(RELEASE_TARGET))"
+%{ if type == "lib" }
 	@echo "  $(BOLD)publish$(SGR0)     -- Run 'cargo publish --dry-run --target RELEASE_TARGET'"
 	@echo "                 (RELEASE_TARGET=$(RELEASE_TARGET))"
+%{ endif }
```

Comment:
`cargo publish` will generate a crate which is normally pushed to crates.io, but we're using `--dry-run` for now as we need to decide on the registry we want to use first.